x/distribution/keeper/validator.go

```diff
@@ -145,7 +145,10 @@ func (k Keeper) updateValidatorSlashFraction(ctx context.Context, valAddr sdk.Va
 	}
 
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
-	val := k.stakingKeeper.Validator(sdkCtx, valAddr)
+	val, err := k.stakingKeeper.Validator(sdkCtx, valAddr)
```

Comment:
Why is it using here a sdk context? While just above (x/distribution/keeper/msg_server.go) it doesn't.
Btw, I have seen this in the codebase multiple times (https://github.com/cosmos/cosmos-sdk/commit/7fef9e91ec93ce5936cb76842fd50dc83196aa67), so we should watch out not to do unecessary unwrapping.